routers/api/v1/repo/pull_review.go

```diff
@@ -886,7 +886,7 @@ func dismissReview(ctx *context.APIContext, msg string, isDismiss bool) {
 		return
 	}
 
-	_, err := pull_service.DismissReview(ctx, review.ID, ctx.Repo.Repository.ID, msg, ctx.Doer, isDismiss)
+	_, err := pull_service.DismissReview(ctx, review.ID, ctx.Repo.Repository.ID, msg, ctx.Doer, isDismiss, false)
```

Comment:
Why API is `false` but web is `true`?

Reply:
Yes else it would be a breaking api change ...

What we could do is to expose it as option at api endpoint